examples/analytics.amp.html

```diff
@@ -33,11 +34,22 @@
   "triggers": {
     "defaultPageview": {
       "on": "visible",
-      "request": "event",
+      "request": "pageview",
       "vars": {
         "eventName": "page-loaded",
         "eventId": "42"
       }
+    },
+    "scrollPings": {
+      "on": "scroll",
+      "request": "event",
+      "scroll-spec": {
```

Comment:
Could you document this. It is very hard to review without exactly understanding what this is actually doing.